torch/onnx/__init__.py

```diff
@@ -264,7 +272,11 @@ def forward(self, x):
             initializers (typically corresponding to parameters) in the
             exported graph will also be added as inputs to the graph. If False,
             then initializers are not added as inputs to the graph, and only
-            the non-parameter inputs are added as inputs.
+            the non-parameter inputs are added as inputs. In such case, no matter
```

Comment:
Consider:
"In such cases, the optimization that might adjust graph inputs will be skipped (e.g., fusing Conv and BatchNorm ops)."

Reply:
Update accordingly to emphasize this will happen only when keep_initializers_as_inputs=False.